src/app/map/map.component.ts

```diff
@@ -4,7 +4,7 @@ import { MatDialogModule } from '@angular/material/dialog';
 import { MatSnackBar } from '@angular/material/snack-bar';
 import { MatChipsModule } from '@angular/material/chips';
 import { MatChipList, MatChipInputEvent } from '@angular/material/chips';
-import { COMMA, ENTER } from '@angular/cdk/keycodes';
+import { COMMA, ENTER, I } from '@angular/cdk/keycodes';

```

Comment:
```suggestion
import { COMMA, ENTER } from '@angular/cdk/keycodes';

```
Doesn't look like the `I` keycode is actually being used?